app/src/_map/tracks/tracks.selectors.js

```diff
@@ -0,0 +1,107 @@
+import { createSelector } from 'reselect';
+import { getTemporalExtent } from '../module/module.selectors.js';
+
+export const getTracksData = state => state.map.tracks.data;
+
+const filterGeojsonByTimerange = (geojson, { start, end }) => {
+  if (!geojson || !geojson.features) return null;
+  const featuresFiltered = geojson.features.reduce((acc, feature) => {
```

Comment:
Is it possible to rename `acc` to something more meaningful?

Reply:
Good catch, fixed here: https://github.com/GlobalFishingWatch/map-client/commit/15eb1b14930dc18c83d63e09fa17cfff1887a33a